src/models/user.rs

```diff
@@ -50,12 +64,11 @@ impl User {
             token,
         }
     }
-
-    pub fn to_profile(self, following: bool) -> Profile {
+    pub fn to_profile(&self, following: bool) -> Profile {
```

Comment:
When this method have `self` as first argument, this warning is displayed by [clippy](https://github.com/rust-lang/rust-clippy): [Imgur](https://i.imgur.com/0aahORt.png)

Reply:
Yes, that's why I suggested using name `into_profile`, because by convention it consumes the receiver. Check this: https://rust-lang-nursery.github.io/api-guidelines/naming.html

Reply:
Thank you very much, it's like that